### ISSUE TITLE
Correct documentation for the tree module

### ISSUE THREAD

**Issue by PSSF23** [online-ml/river#636]
## Versions

**river version**: 0.7.1
**Python version**: 3.8
**Operating system**: Mac Big Sur

## Describe the bug

On the document [page](https://riverml.xyz/latest/api/tree/HoeffdingTreeClassifier/) for `HoeffdingTreeClassifier`, the `predict_many` function is listed, but actually calling it would result in `NotImplementedError`.
## Steps/code to reproduce

```python
ht = tree.HoeffdingTreeClassifier()
idx = range(1024)
X = dict(zip(idx, idx))
y = 1
ht.learn_one(X, y)
ht.predict_many(X)
```

**Comment by smastelini** [comment]
Hi @PSSF23, you're right. The tree module still doesn't support mini-batch updates.

@MaxHalford, is there an easy way to fix the documentation, other than changing the class hierarchy?

**Comment by PSSF23** [comment]
@smastelini In addition, `HoeffdingTreeClassifier` has no `depth` attributes. This might apply to all tree models as well.
```python
AttributeError: 'HoeffdingTreeClassifier' object has no attribute 'depth'
```

**Comment by smastelini** [comment]
Hi @PSSF23. If you're using the development version, this parameter was renamed to `height`. This was done to reflect the correct tree data structure naming convention.

**Comment by MaxHalford** [comment]
Fixed in #666 :)
